tests/dualtor/test_toggle_mux.py

```diff
@@ -27,25 +29,53 @@ def restore_mux_auto_mode(duthosts):
     duthosts.shell('config muxcable mode auto all')
 
 
+@pytest.fixture(scope="module")
+def get_interval_v4(duthosts):
+    mux_linkmgr_output = duthosts.shell('sonic-cfggen -d --var-json MUX_LINKMGR')
+    mux_linkmgr = list(mux_linkmgr_output.values())[0]['stdout']
+    if len(mux_linkmgr) != 0:
+        Cur_interval_v4 = json.loads(mux_linkmgr)['LINK_PROBER']['interval_v4']
+        return Cur_interval_v4
+    else:
+        return None
+
+
 @pytest.mark.parametrize("active_side", [UPPER_TOR, LOWER_TOR])
-def test_toggle_mux_from_simulator(duthosts, active_side, toggle_all_simulator_ports, get_mux_status, restore_mux_auto_mode):
+def test_toggle_mux_from_simulator(duthosts, active_side, toggle_all_simulator_ports,
+                                   get_mux_status, get_interval_v4, restore_mux_auto_mode):
     logger.info('Set all muxcable to manual mode on all ToRs')
     duthosts.shell('config muxcable mode manual all')
 
+    Cur_interval_v4 = get_interval_v4
+    if Cur_interval_v4 is not None:
+        duthosts.shell('sonic-db-cli CONFIG_DB HSET "MUX_LINKMGR|LINK_PROBER" "interval_v4" "{}"'
+                       .format(DEFAUL_INTERVAL_V4))
+        duthosts.shell('docker exec -i mux supervisorctl restart linkmgrd')
+
     logger.info('Toggle mux active side from mux simulator')
     toggle_all_simulator_ports(active_side)
 
     check_result = wait_until(60, 5, 2, check_mux_status, duthosts, active_side)
 
+    if Cur_interval_v4 is not None:
+        duthosts.shell('sonic-db-cli CONFIG_DB HSET "MUX_LINKMGR|LINK_PROBER" "interval_v4" "{}"'
+                       .format(Cur_interval_v4))
+        duthosts.shell('docker exec -i mux supervisorctl restart linkmgrd')
+
```

Comment:
Just a non-mandatory suggestion: since this part of code is repeated a couple of times, can we put it into `dual_tor_utils`? Maybe a function named `update_linkmgrd_probe_interval`. 

Similar to the one below:
https://github.com/sonic-net/sonic-mgmt/blob/7c49983233f2e6643925dd698b435523eaaad993/tests/common/dualtor/dual_tor_utils.py#L1341

Reply:
Sure, that would be better